src/integration-test/java/com/mongodb/jdbc/integration/testharness/models/TestEntry.java

```diff
@@ -0,0 +1,31 @@
+package com.mongodb.jdbc.integration.testharness.models;
+
+import java.util.List;
+
+public class TestEntry {
+    public String description;
+    public String sql;
+    public String skip_reason;
+    public Integer row_count;
+    public Boolean rowcount_gte;
```

Comment:
```suggestion
    public Integer row_count;
    public Boolean row_count_gte;
```
[**super**-nit] Didn't catch this in the google doc, but should/could we make these two fields consistent by either including the `_` between `row` and `count` in both, or not including it in either? 😅